src/components/account_overview/AccountOverview.stories.mdx

```diff
@@ -66,6 +68,10 @@ export const Default = ({
 
 #### Custom
 
+This example leverages the property object of each AccountOverview detail. Notice that the `label` name,
```

Comment:
We can tighten this writing up. Combine the first 2 sentences into.

"You can also configure each AccountOverview detail property with an object to customize the property's `label`, `size`, and `value` for example." 

Good mention of implicit behavior.